app/utils/report/test.py

```diff
@@ -119,8 +119,6 @@ def create_test_report(data, email_format, db_options,
 
     test_group_docs = list(utils.db.find(
         database[models.TEST_GROUP_COLLECTION],
-        100,
```

Comment:
Did you try passing zero instead? Can't remember which version of the python driver it's in use, but I think by default the underlying library uses 0 for no limit, same as the `skip` parameter.

Reply:
I did try with a 0 limit and got... 0 results back :)

Reply:
Hrm I tried again on my local set-up and now got unlimited results.  I'll see if it's something that changed in the python driver.

Reply:
I checked on pymongo docs and code, and that's definitely weird then.

The underlying class (`Cursor`) powering the `find` function sets `limit` to 0 by default as a "no limit set" value. Theoretically passing 0 should return all results...

Reply:
OK I'll verify that again on staging and confirm.  Maybe it was a coincidence when I got 0 results.